app/models/subject.rb

```diff
@@ -47,5 +48,9 @@ def hidden_by_default?
     revalid? || inactive? || outside_montevideo? || extension_module?
   end
 
+  def update_rating
+    update(average_rating: reviews.average(:rating))
```

Comment:
```suggestion
    update!(average_rating: reviews.average(:rating))
```

should we use the bang version here?